decidim-core/app/cells/decidim/data_consent/dialog.erb

```diff
@@ -0,0 +1,29 @@
+<div id="cc-dialog-wrapper" class="flex-center hide" role="region">
```

Comment:
Let's also do the following changes to this as per the discussion with @andreslucena :
```suggestion
<div id="cc-dialog-wrapper" class="flex-center hide robots-noindex robots-nocontent noindex" data-nosnippet role="region">
```